doc/arch/adr-002-delete-unpublished-documents-from-index.md

```diff
@@ -0,0 +1,90 @@
+# Decision record: delete documents from the govuk search index when an unpublishing message is received
```

Comment:
I feel that something like "Handling unpublishing edge case" is more in line with what we are discussing and would allow us to update this when we handle the current delete case in a nicer fashion.